### DISCUSSION TITLE
Style using the fast method.

### DISCUSSION THREAD

**Discussion by DarkAlchy** [TheLastBen/fast-stable-diffusion#276] [General]
Any guidance on how to do it?  How would we call it in the prompt and how many images required, and the steps?  I know using the older method it took a boat load of steps to make a style (far more than to make a model).

**Comment by TheLastBen** [comment]
try the new method with 15 pics and 2500 steps.
make sure you name the images something like this : https://i.imgur.com/d2lD3rz.jpeg

**Reply by DarkAlchy** [reply]
> try the new method with 15 pics and 2500 steps. make sure you name the images something like this : https://i.imgur.com/d2lD3rz.jpeg

How would I call it as a style?

**Reply by TheLastBen** [reply]
just use a random identifier, like "oingngern". then in A1111 you summon it using : "in in the style of oingngern" use () for weighing like "a house detailed the style of (oingngern)"

**Reply by DarkAlchy** [reply]
> just use a random identifier, like "oingngern". then in A1111 you summon it using : "in in the style of oingngern" use () for weighing like "a house detailed the style of (oingngern)"

Perfect.  Thanks again.
